server/src/main/java/org/apache/iotdb/db/protocol/influxdb/function/selector/InfluxFirstFunction.java

```diff
@@ -45,10 +45,7 @@ public InfluxFunctionValue calculateByIoTDBFunc() {
 
   @Override
   public void updateValueIoTDBFunc(InfluxFunctionValue... functionValues) {
-    if (value == null && getTimestamp() == null) {
-      value = functionValues[0].getValue();
-      setTimestamp(functionValues[0].getTimestamp());
-    } else if (getTimestamp() < functionValues[0].getTimestamp()) {
+    if (functionValues[0].getTimestamp() < getTimestamp()) {
```

Comment:
If it is the first time to enter the updateValueIoTDBFunc, the result of getTimestamp should be NULL, Will this cause NullPointerException?